src/main/java/Duke.java

```diff
@@ -6,5 +7,42 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can i do for you?");
+        String[] List = new String[100];
+        int ListNo = 0;
+        do{
+            line = in.nextLine();
+            if (line.equals("bye")){
+                break;
+            }else if(line.compareTo("list") != 0) {
```

Comment:
Spacing is an issue in this piece of code, not just for the if-else block as mentioned by chydarren